api/templates/financial.xml

```diff
@@ -1,9 +1,13 @@
 <FinancialRecord Type="Pooled" Version="1">
-  {{tmpl "financial-bankruptcy.xml" .Financial.Bankruptcy}}
-  {{tmpl "financial-card.xml" .Financial.Card}}
-  {{tmpl "financial-credit.xml" .Financial.Credit}}
+  {{ if formType "SF86,SF85P" }}
```

Comment:
Something I'm mulling over (i.e., not asking for a change) – would it be better off for the individual template files to have the `if` block. If you consider them objects/components, the knowledge of where they should be used is maintained at that level, rather than its parent. 

In particular, cases like `financial-credit.xml` where there is an `SF86` conditional would be more clear, as you could just see in that same file that it was already restricted to the `SF86` and `SF85P`.

Reply:
I tried this, there's enough places where XML is being inserted at the top level (see foreign.xml for instance) that trying to put the rules in the sub-sections ends up being redundant. I think I'm going to leave things as is rather than making it so sometimes you have to check the top level and sometimes you don't.